webpack.config.js

```diff
@@ -42,17 +43,31 @@ function afterEmit(root, functions, provider) {
   return {
     apply: (compiler) => {
       compiler.hooks.afterEmit.tap('Functions Deployment', async () => {
-        Promise.all(
-          functions.map((item) =>
-            item.filename.endsWith('.zip') && provider.createOrUpdateFunction(path.parse(item.filename).name, item.content).catch((x) => console.error(x))
-          )
-        ).then(() => {
-          const redirects = {}
-          // compilation.entrypoints.forEach((_, key) => (redirect[filename] = createRedirect(filename)))
-          // fs.writeFileSync(resolve(root, 'public', 'functions-redirects.json', `${JSON.stringify(redirects)}\n`)
-        })
+        const urls = await Promise.all(
+          functions.map(async (item) => {
+            if (!item.filename.endsWith('.zip')) {
+              return
+            }
+
+            const functionName = `sfj-${path.parse(item.filename).name}`
+
+            return [
+              functionName,
+              await provider.createOrUpdateFunction(functionName, item.content),
+            ]
+          })
+        )
+
+        const functionsURLs = urls
+          .filter(item => item && item[1] !== undefined)
+          .reduce((acc, curr) => ({...acc, [curr[0]]: curr[1]}), {})
+
+        fs.writeFileSync(
+          path.resolve(root, 'public', 'functions-redirects.json'),
```

Comment:
Having a constants.ts file with constants like the file name, or maybe the whole path, is a good practice